ansible-scylla-node/tasks/upgrade/upgrade_debian10.yml

```diff
@@ -0,0 +1,44 @@
+---
+
+# Add Scylla apt key
+- name: Add Scylla apt key
+  ansible.builtin.apt_key:
+    keyserver: hkp://keyserver.ubuntu.com:80
+    id: 5e08fbd8b5d6ec9c
```

Comment:
we have a variable for that, haven't we?

Reply:
This depends of the OS, using the same variable for the whole "Debian" family is a mistake.